context/llpcCompiler.h

```diff
@@ -261,6 +311,38 @@ class Compiler: public ICompiler
     static void TranslateSpirvToLlvm(const PipelineShaderInfo*    pShaderInfo,
                                      llvm::Module*                pModule);
 
+#if LLPC_CLIENT_INTERFACE_MAJOR_VERSION < 38
+    ShaderEntryState LookUpShaderCaches(IShaderCache*       pAppPipelineCache,
+                                        MetroHash::Hash*    pCacheHash,
+                                        BinaryData*         pElfBin,
+                                        ShaderCache**       ppShaderCache,
+                                        CacheEntryHandle*   phEntry);
+
+    static void UpdateShaderCaches(bool                bInsert,
```

Comment:
bInsert -> insert

Reply:
Again, it was like that before my change. :-) But since I'm changing that line anyway, I'll fix it.